content/en/runtime.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Runtime
+status: Feedback Appreciated
+category: concept
+tags: ["application", "", ""]
+---
+
+## What it is
+
+A runtime, in general, executes a piece of software.
+It is an abstraction of the underlying operating system that knows how to translate the program's commands into respective actions on the operating system level. 
+
+In the context of [cloud native](/cloud-native-apps/), _runtime_ generally refers to container runtime. 
+A container runtime specifically implements the [Open Container Initiative](https://opencontainers.org/) specification to ensure consistent handling around the ecosystem. 
+This includes the container image specification, the method by which the runtime can retrieve an image, 
+and how that image is unpacked, layered, mounted, and executed. 
+These runtimes utilize kernel features such as namespaces and cgroups to provide isolation and resource management for the containers.
+
+## Problem it addresses
+
+Container runtimes are a necessary component of container orchestrators such as Kubernetes. 
+The runtime is scoped to only handle the lifecycle of the containers. 
+User-facing features, such as scaling and service discovery, fault tolerance, etc., are managed by the orchestrator or some other high-level tool that interfaces with the runtime.
+
+Without a container runtime, the user would have to manually handle the container lifecycle, interface directly with cgroups and namespaces in the kernel, and handle hardware resources.
+The container runtime acts as an abstraction layer for these features.
+
+## How it helps
+
+Container runtimes help to address this by taking care of all those low-level actions for us; 
+thus, they are responsible for launching, managing, and terminating containers.
```

Comment:
Is this part of container lifecycle management? If so, we should list it above where the term in introduced.

Reply:
apologies for this one. will add in the further commit

Reply:
would suggest to delete this sentence because we talk about it in this section saying that the runtime is responsible for the execution of containers.  
```suggestion
```